godtools/App/Features/ToolDetails/Data-DomainInterface/GetToolDetailsRepository.swift

```diff
@@ -47,8 +47,17 @@ class GetToolDetailsRepository: GetToolDetailsRepositoryInterface {
             return Just(noToolDomainModel)
                 .eraseToAnyPublisher()
         }
-        
-        guard let translation = translationsRepository.getLatestTranslation(resourceId: toolId, languageCode: translateInLanguage) else {
+    
+        let translation: TranslationModel
+        if let appLanguagetranslation = translationsRepository.getLatestTranslation(resourceId: toolId, languageCode: translateInLanguage) {
+            
+            translation = appLanguagetranslation
+        }
+        else if let defaultTranslation = translationsRepository.getLatestTranslation(resourceId: toolId, languageCode: toolDataModel.attrDefaultLocale) {
+            
+            translation = defaultTranslation
```

Comment:
Hey @rachaelblue I'm not 100% certain on this here.  The Jira task wasn't very clear on what needed to display in the default locale vs app language.

I noticed the about description text is displayed in the default locale and the versions description text is in the app language.  I wonder if that is intended?

Also the tool name looks to be displaying in English in the version card.  There could be a bug here and I'm curious what the language translation requirements are for the version card interface strings.  I'm assuming it would be similar to the tool lists, the name and description would display in the app language if supported, otherwise fallback to the default locale.

![tool-details-2](https://github.com/CruGlobal/godtools-swift/assets/59846460/69737154-56bd-4d28-9727-15da7f5714c4)

![tool-details-3](https://github.com/CruGlobal/godtools-swift/assets/59846460/b7151ce3-8b2c-4760-bac2-07149d73e5d6)

Reply:
Hey @levieggertcru, Jill confirmed that only non-interface strings should be translated into the default language (she circled in red).  On the versions tab, the description is an interface string, so it should be in app language.
![img_4238_720](https://github.com/CruGlobal/godtools-swift/assets/8647064/2ecb98f9-d498-4ccf-8277-4922691ed8a9)

I did update the translation logic on the versions tool card title since it was still defaulting to English.

<img src="https://github.com/CruGlobal/godtools-swift/assets/8647064/a09cdf24-756f-4f4d-92f0-f76bb8e2d8de" width=250>